pkg/util/filewatcher/filewatcher.go

```diff
@@ -0,0 +1,34 @@
+package filewatcher
```

Comment:
This package doesn't seem to be adding much value that isn't already present in the `fsnotify` package. Could the `tokenfile` package just use `fsnotify` directly?